cdap-runtime-ext-dataproc/src/main/resources/gcp-dataproc.json

```diff
@@ -257,6 +266,20 @@
         }
       ]
     },
+    {
+      "label": "GCP CMEK Information",
+      "properties": [
+        {
+          "widget-type": "textbox",
+          "label": "Encryption Key Name",
+          "name": "encryptionKeyName",
+          "description": "The GCP customer managed encryption key(CMEK) name used by Cloud Dataproc",
+          "widget-attributes": {
+            "placeholder": "Specify your Encryption Key Name"
```

Comment:
can you put an example value in the placeholder? Something like `/projects/<your-project>/...`